tests/wallet-automation/Earthfile

```diff
@@ -0,0 +1,74 @@
+VERSION 0.7
+
+deps:
+    FROM mcr.microsoft.com/playwright:v1.41.0-jammy
+    WORKDIR /app
+
+    RUN apt-get update && apt-get -y install libnss3 libatk-bridge2.0-0 libdrm-dev libxkbcommon-dev libgbm-dev libasound-dev libatspi2.0-0 libxshmfence-dev
+    RUN apt-get install -y postgresql-client
+    RUN apt-get update && apt-get install -y xvfb
+
+
+    #Java is needed for allure
+    # RUN apt-get update && \
+    #     apt-get install -y openjdk-8-jdk && \
+    #     apt-get install -y ant && \
+    #     apt-get clean && \
+    #     rm -rf /var/lib/apt/lists/ && \
+    #     rm -rf /var/cache/oracle-jdk8-installer;
+    # ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
+
+    COPY package.json .
+    COPY package-lock.json .
+
+    RUN npm install
+    ENV PATH /app/node_modules/.bin:$PATH
+
+src:
+    FROM +deps
```

Comment:
In the `src` target we want to copy all the source files needed for the test
So the  `COPY . .` command in the `test` target should go in here.
Also you dont want to copy everything but you want to specify the files/directory you need to run the tests. In this case for example `COPY --dir typhon .`